### DISCUSSION TITLE
URL to new-labels.zip?

### DISCUSSION THREAD

**Discussion by EdMailletUnum** [treeverse/dvc#9986] [Help] [answered]
Does anyone have a url to download the new-labels.zip from the tutorials that can be used to download in a browser?

I'm stuck in proxy hell with dvc get and can't get the 
dvc get https://github.com/iterative/dataset-registry tutorials/versioning/new-labels.zip
to work. 

I just want to finish the tutorial. I found the data.zip url in another post but can't find the new-labels.zip.

**Comment by dberenbaum** [comment, marked as answer]
You can get the URL with this command (in case that doesn't work for you, the URL is shown below):

```
$ dvc get --show-url https://github.com/iterative/dataset-registry tutorials/versioning/new-labels.zip
https://remote.dvc.org/docs/dvc/2e/aa473159443e75e6fb7b29e56c0787
```

**Reply by EdMailletUnum** [reply]
Many thanks! FYI. I did manage to get the dvc command to work on a different VM and our proxy. However, the commands:
dvc get https://github.com/iterative/dataset-registry tutorials/versioning/data.zip
dvc get https://github.com/iterative/dataset-registry tutorials/versioning/new-labels.zip

can't find the files. Here's the message: (Ironcially, the dvc list will list them)
FileNotFoundError: [Errno 2] No storage files available: 'tutorials/versioning/new-labels.zip'

2023-09-29 15:55:02,619 DEBUG: Version info for developers:
DVC version: 3.23.0 (pip)
-------------------------
Platform: Python 3.8.10 on Linux-5.10.16.3-microsoft-standard-WSL2-x86_64-with-glibc2.29
Subprojects:
        dvc_data = 2.16.4
        dvc_objects = 1.0.1
        dvc_render = 0.6.0
        dvc_task = 0.3.0
        scmrepo = 1.3.1
Supports:
        http (aiohttp = 3.8.5, aiohttp-retry = 2.8.3),
        https (aiohttp = 3.8.5, aiohttp-retry = 2.8.3)
Config:
        Global: /home/ubuntu/.config/dvc
        System: /etc/xdg/dvc
